app/views/attendances/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<%= simple_form_for(@attendance) do |f| %>
+  <%= f.error_notification %>
+  <%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
```

Comment:
- [ ] use a helper method to render the if statement in this line there should be no if statements within views or partials